content/html/concepts/semantic-html/terms/rb/rb.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '<rb>'
+Description: 'Serving as the base component in ruby annotations for East Asian text, this element's use has declined and is now listed as deprecated.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<rb>`** element designates the base text for ruby annotations in East Asian languages. It must be within `<ruby>` tags and is often paired with [`<rt>`](https://www.codecademy.com/resources/docs/html/semantic-html/rt) for pronunciation guides and `<rp>` for fallback rendering.
+
+> **Note:** The `<rb>` tag is considered deprecated, meaning it's no longer recommended for use in new projects. Be sure to consult current web standards and best practices when working with ruby annotations.
+
+## Syntax
+
+```html
+<ruby>
+  <rb>Base text goes here</rb>
+  <rp>(</rp> <rt>Annotation text goes here</rt> <rp>)</rp>
+</ruby>
+```
+
+## Example
+
+The example below showcases how the `<rb>` tag is used to transliterate the Chinese characters meaning 'I like to write code' inside a `<ruby>` element.
+
+```html
+<ruby>
+  <rb>我</rb><rt>I</rt> <rb>喜欢</rb><rt>like</rt> <rb>写</rb><rt>to write</rt>
+  <rb>程序</rb><rt>code</rt>
+</ruby>
+```
+
+This is what it looks like in the browser:
+
+![Output of the above code with English translations](https://raw.githubusercontent.com/Codecademy/docs/main/media/iliketowritecode.png)
```

Comment:
[nitpick] I recommend making the image smaller so it can fit into the entry. Here's what I use to help me with that: https://imageresizer.com/
```suggestion
![Output of the above code with English translations](https://raw.githubusercontent.com/Codecademy/docs/main/media/iliketowritecode.png)
```